crates/sui-framework/sources/governance/validator.move

```diff
@@ -53,8 +53,13 @@ module sui::validator {
     /// Invalidworker_address field in ValidatorMetadata
     const EMetadataInvalidWorkerAddr: u64 = 7;
 
+    /// Commission rate set by the validator is higher than the threshold
+    const ECommissionRateTooHigh: u64 = 8;
+
     const EInvalidProofOfPossession: u64 = 0;
 
+    const MaxCommissionRate: u64 = 10_000; // Max rate is 100%, which is 10K base points
```

Comment:
For this type of constant we normally use all caps in other places in the framework, such as here: https://github.com/MystenLabs/sui/blob/a91702ba32795ae247343987bf832d866842e4fd/crates/sui-framework/sources/governance/voting_power.move#L29
Change the naming here to be consistent?